Resources/Prototypes/ai_factions.yml

```
@@ -177,4 +183,14 @@
   - Heretic # goob edit - heretics
   - Blob # Goobstation - Blob
   - Wizard # goob edit - wizard
-  - PirateFaction # goob - pirates
\ No newline at end of file
+  - PirateFaction # goob - pirates
+  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)